### DISCUSSION TITLE
Default language setting

### DISCUSSION THREAD

**Discussion by SamKr** [jordan-dalby/ByteStash#101] [Ideas]
For starters: this is GREAT. I've been waiting for something like this! The current solutions are either expensive, not what I want, bloated or most of the time all of the above. This is looking like exactly what I hoped for. Good job and thank you so much for sharing ❤️ 

Since I use C# in 80% of the times, I'd love to be able to select a default language. Every click or keystroke saved always feels good 😁 

Again, thanks for your work :)

PS. Personally I love the logo, it's refreshing after all the bland lookalikes these days.

**Comment by jordan-dalby** [comment]
@SamKr 
Thanks for the kind words!

Good idea, there's an issue in place for revamping the languages dropdown, I think this could be included. #51 

PS. Thanks :)
